bin/download_obsid_caldb

```diff
@@ -463,7 +462,8 @@ def get_tools(infile):
                ### 'rmf_file' : None,  # ???
                ### 'gain_file' : None # ???
                }
-    mkacisrmf = {'sc_matrix' : None, }  # infile
+    temps = ",".join([str(x) for x in range(153,178,2)])
```

Comment:
I assume your complaint is that there's no way to automate creating the list of temperatures to use here, soit has to be hard-coded?

Reply:
I had hoped I could set `fp_temp=None` or _blank_ or `""` or even `delattr` it to remove any constraint; but none of those worked.

Honestly I'm shocked that a comma separated list works (or as it turns colons work too, who knew?).  Just a bit of maintenance PITA if the range changes and/or the granularity (step size) changes.